Customer.io project/components/useTheme.tsx

```
@@ -0,0 +1,75 @@
+  );
+};
+
+// Theme Toggle Button Component
+export const ThemeToggleButton: React.FC = () => {
+  const { theme, toggleTheme } = useTheme();
+
+  return (
+    <button
+      onClick={toggleTheme}
+      className="p-2 hover:bg-gray-200 dark:bg-gray-700/30 hover:dark:bg-gray-700 rounded-full transition-all"
+    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)